inc/saivlan.h

```diff
@@ -281,6 +281,18 @@ typedef enum _sai_vlan_attr_t
     /** Custom range base value */
     SAI_VLAN_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
+    /**
+     * @brief IGMP Snooping enable or disable control for VLAN
+     *
+     * IGMP Snooping enable control for VLAN. Default is
+     * disabled
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_VLAN_ATTR_CUSTOM_IGMP_SNOOPING_ENABLE,
```

Comment:
@kcudnik Build is failing when I add it in the custom range with "sai_vlan_attr_t: values are not increasing by 1: last: 14 current: 268435457". Can you please help to fix this?